tools/github-team-user-store/GitHubTeamUserStore/GitHubTeamUserStore/Constants/ProductAndTeamConstants.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GitHubTeamUserStore.Constants
+{
```

Comment:
```suggestion

```

Since .net 6, you can reduce the file indentation everywhere by using [file-scoped namespaces](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-10.0/file-scoped-namespaces).

Reply:
@konrad-jamrozik, this is a preference. I prefer the namespace in the file otherwise I'd select the checkbox to use file-scoped namespaces.